recipes/lstmcpipe/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "lstmcpipe" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lstmcpipe-{{ version }}.tar.gz
+  sha256: 0ebedd5211a190487ba6b6702679d310703cd8aa724bad5d802299dab0aba397
+
+build:
+  entry_points:
+    - lstmcpipe = lstmcpipe.lstmcpipe_start:main
+    - lstmcpipe_plot_models_importance = lstmcpipe.plots.plot_models_importance:main
+    - lstmcpipe_plot_irfs = lstmcpipe.plots.plot_irfs:main
+    - lstmcpipe_hiperta_r0_to_dl1lstchain = lstmcpipe.hiperta.hiperta_r0_to_dl1lstchain:main
+    - lstmcpipe_dl2_to_sensitivity = lstmcpipe.scripts.script_dl2_to_sensitivity:main
+    - lstmcpipe_train_test_split = lstmcpipe.scripts.script_train_test_splitting:main
+    - lstmcpipe_lst_core_r0_dl1 = lstmcpipe.scripts.script_batch_filelist_lst:main
+    - lstmcpipe_lst_core_dl1ab = lstmcpipe.scripts.script_batch_filelist_lst_dl1ab:main
+    - lstmcpipe_cta_core_r0_dl1 = lstmcpipe.scripts.script_batch_filelist_cta:main
+    - lstmcpipe_rta_core_r0_dl1 = lstmcpipe.scripts.script_batch_filelist_rta:main
+    - lstmcpipe_compare_irfs = lstmcpipe.scripts.script_compare_irfs:main
+    - lstmcpipe_validate_config = lstmcpipe.scripts.script_lstmcpipe_validate_config:main
+    - lstmcpipe_generate_config = lstmcpipe.scripts.lstmcpipe_generate_config:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.9
+    - lstchain >=0.10.0
```

Looks like the metadata is:

```suggestion
    - lstchain >=0.9.0
```

any reason to be more restrivtive?

---

lstchain has no conda package before 0.10.0

---

That should not be a problem b/c 0.10.0 will be found and, if folks ever want to use this recipe outside of conda-forge with 0.9.0, they don't need to dig into the metadata to fix it.

---

I see, thanks, I fixed it